### ISSUE TITLE
Adding new search engines is not working

### ISSUE THREAD

**Issue by smadurange** [atlas-engineer/nyxt#2973]
I'm using version 3.0.0. I created a configuration file ~/.config/nyxt/config.lisp and here's my full configuration file:

```
(define-configuration buffer
  ((default-modes
    (pushnew 'nyxt/mode/vi:vi-normal-mode %slot-value%))))

(defvar *my-search-engines*
  (list
   '("google" "https://google.com/search?q=~a" "https://google.com")
   '("python3" "https://docs.python.org/3/search.html?q=~a"
     "https://docs.python.org/3")
   '("doi" "https://dx.doi.org/~a" "https://dx.doi.org/"))
  List of search engines.)

(define-configuration context-buffer
  Go through the search engines above and make-search-engine out of them.
  ((search-engines
    (append
     (mapcar (lambda (engine) (apply 'make-search-engine engine))
             *my-search-engines*)
     %slot-default%))))
```
I got the add search engine config straight from [documentation](https://nyxt.atlas.engineer/documentation#search-engines). When I launch the browser, I see the following error message:

```
Warning: Error while parsing arguments to DEFMACRO DEFVAR: too many elements in (*MY-SEARCH-ENGINES* (LIST '("google" "https://google.com/search?q=~a" "https://google.com") '("python3" "https://docs.python.org/3/search.html?q=~a" "https://docs.python.org/3") '("doi" "https://dx.doi.org/~a" "https://dx.doi.org/")) LIST OF SEARCH ENGINES.) to satisfy lambda list (VAR &OPTIONAL (VAL) (DOC)): between 1 and 3 expected, but got 6
```

**Comment by progfolio** [comment]
The docstring needs to be in quotes:

```lisp
"List of search engines."
```

Otherwise, each word in that sentence is being interpreted as a separate (symbol) argument to defvar.

**Comment by aartaka** [comment]
@aadcg, there actually was a problem with docstring displaying improperly in the manual, so the example snippet was broken. It's no OP's fault here :P

**Comment by aartaka** [comment]
@smadurange, 678b78c01684965815b9381194b15a1f3ceb0315 should fix it. It'll be included in the next (supposedly a close 3.0.1) release!
